src/sidebars/search/search-controller.js

```diff
@@ -21,7 +21,9 @@
         }
 
         select(stock) {
-            this.selectionService.select(stock);
+            if (stock.favourite) {
+                this.selectionService.select(stock);
+            }
```

Comment:
Actually I may have misinterpretted the ticket. Perhaps selection should just be disabled across the board in compact. Easily done.